streaming/base/storage/download.py

```diff
@@ -422,6 +423,47 @@ def download_from_dbfs(remote: str, local: str) -> None:
     os.rename(local_tmp, local)
 
 
+def download_from_alipan(remote: str, local: str) -> None:
+    """Download a file from remote Alipan to local.
+
+    Args:
+        remote (str): Remote path (Alipan).
+        local (str): Local path (local filesystem).
+    """
+    from alipcs_py.alipcs import AliPCSApiMix, AliPCSError
+    from alipcs_py.commands.download import download
+
+    web_refresh_token = os.environ['ALIPAN_WEB_REFRESH_TOKEN']
+    web_token_type = 'Bearer'
+    alipan_encrypt_password = os.environ.get('ALIPAN_ENCRYPT_PASSWORD', '').encode()
+
+    api = AliPCSApiMix(web_refresh_token, web_token_type=web_token_type)
+
+    obj = urllib.parse.urlparse(remote)
+    if obj.scheme != 'alipan':
+        raise ValueError(
+            f'Expected obj.scheme to be `alipan`, instead, got {obj.scheme} for remote={remote}')
+    if obj.netloc != '':
+        raise ValueError(
+            f'Expected remote is like alipan:///path/to/some, instead, got remote={remote}')
+
+    remote_path = pathlib.Path(obj.path)
+    filename = remote_path.name
+    localdir = pathlib.Path(local).parent
+
+    try:
```

Comment:
Wondering, what's the point of `try..except` if you are just raising the same exception in the `except` block?

Reply:
A `FileNotFoundError` should be raised here.

Reply:
done.